eclipsePlugin/build.gradle

```diff
@@ -41,9 +41,9 @@ dependencies {
   compile fileTree(dir:localProps.getProperty('eclipsePlugin.dir'), include:'**/*.jar',
     exclude:['**/datanucleus-enhancer*.jar', 'edu.umd.cs.findbugs.**/*.jar'])
 
-  includedLibs(project(path:':findbugs', configuration:'annotations')) {
-    transitive = false
-  }
+  includedLibs project(':spotbugs-annotations')
+  includedLibs 'com.google.code.findbugs:jsr305:3.0.1'
```

Comment:
if you do this, you should probably remove `jsr305.jar` from the libs folder, remove it from line 31 of this build script, and change the manifest accordingly.